tests/test_cadquery.py

```diff
@@ -24,6 +24,9 @@
     makeCube,
 )
 
+# test data directory
+testdataDir = os.path.join(os.path.dirname(__file__), "testdata")
```

Comment:
Why isn't `tempfile.gettempdir()` from line 31 being used here?

https://github.com/CadQuery/cadquery/blob/a947d0b4be17c76dba0c25c79352bd8e6f572b26/tests/test_cadquery.py#L31

Reply:
If there's a good argument for not using tempfile here, I'm open to hearing it.

Reply:
I'm not against any changes you want to make. I just modified an existing test.

I don't know how to load the font file in the "testdata" directory from the tempfile OUTDIR path, so I'm not sure what to change.

Reply:
Ok, I think it's fine as-is then.